spec/2019-11-15-schnorrmultisig.md

```diff
@@ -62,12 +59,13 @@ The new mode operates by checking public keys against signatures, according to t
     Set a cursor on the last signature.
     Set another cursor on the last public key.
     Calculate scriptCode.
-    If activated, and all pubkeys are correctly encoded, and all signatures are either null or correctly encoded Schnorr, then:
+    If activated, and the checkbits element is not null, then:
         # New mode (2)
-        Convert the dummy element to integer, called checkbits; if it was not minimally encoded, fail script.
+        Convert the checkbits element to integer, called checkbits; if it was not minimally encoded, fail script.
         If checkbits < 0, fail script.
         Loop while the signature and key cursors are not depleted:
             If the least significant bit of checkbits is 1, then:
+                If the current pubkey is not correctly encoded, fail script.
```

Comment:
also signature encoding is checked (the 64 byte schnorr is always good, but the hashtype byte needs to be checked).

Reply:
OK, will ad that.